src/components/MessageList.tsx

```diff
@@ -0,0 +1,448 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/** ****************************************************************************
+ * Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import React, { useState, useEffect } from 'react';
+import { observer } from 'mobx-react-lite';
+import { useStore } from '../hooks/useStore';
+import '../styles/message-list.scss';
+import '../styles/splitter.scss';
+import SplitView from '../split-view/SplitView';
+import SplitViewPane from '../split-view/SplitViewPane';
+import { nanoid } from '../../node_modules/nanoid';
+import { downloadFile } from '../helpers/downloadFile';
+
+export type Indicator =
+	| 'indicator-unvisible'
+	| 'indicator-edited'
+	| 'indicator-successful'
+	| 'indicator-unsuccessful';
+
+export interface ParsedMessageItem {
```

Comment:
move ParsedMessageItem, ActMessageItem, isParsedMessageItem, isActMessageItem to models/message